tests/h/views/api/helpers/links_test.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import pytest
+
+from h.views.api.helpers import links
+
+
+class TestRegisterLink(object):
```

Comment:
The previous tests for link registration [still exist](https://github.com/hypothesis/h/blob/fe8ab1cab3131be4be464e47c59dc8bfe46842d2/tests/h/views/api/config_test.py#L108), which means that we're testing some of this twice. The existing test should now just check that the helper is invoked.